src/plugins/intel_gna/src/request/worker_impl.cpp

```diff
@@ -113,5 +127,18 @@ void WorkerImpl::check_if_free() {
     }
 }
 
+void WorkerImpl::cleanup_unfinished_submodules() {
```

Comment:
IMO we should get rid of this function,
otherwise the function name could be:
cleanup_model_subrequests

Reply:
I have modified it.